Add ref to dyncfg (#18793)
netdata/netdata: docs/netdata-agent/configuration/README.md

# Netdata Agent Configuration

> **Info**
>

> > You can also **dynamically configure** any Agent on your Space through Netdata Cloud. Check the [Dynamic Configuration Manager](/docs/netdata-agent/configuration/dynamic-configuration.md) documentation for more details.

Reviewer: ```suggestion
> Netdata Cloud lets you configure Agents on the fly. Check the [Dynamic Configuration Manager](/docs/netdata-agent/configuration/dynamic-configuration.md) documentation for details.
```
